pkg/client/devops/jenkins/credential.go

```diff
@@ -237,6 +237,10 @@ func (j *Jenkins) CreateCredentialInProject(projectId string, credential *v1.Sec
 		return "", err
 	}
 
+	// the jenkins api `/job/{projectId}/credentials/store/folder/domain/_/createCredentials`
```

Comment:
Typo fixing suggestion:

* `jenkins` -> `Jenkins`
* `api` -> `API`

Jenkins does not have `projectId`. Please use `{jobName}` instead.

Reply:
OK